wolfcrypt/test/test.c

```diff
@@ -1767,10 +1767,12 @@ options: [-s max_relative_stack_bytes] [-m max_relative_heap_memory_bytes]\n\
 #endif
 
 #if defined(HAVE_HPKE) && defined(HAVE_ECC) && defined(HAVE_AESGCM)
+    PRIVATE_KEY_UNLOCK();
```

Comment:
Is this the right place to put the unlock? Why not on the specific key use in HPKE?

Reply:
that's the code pattern for all the other tests that need it -- just following the pattern.